server/container_create_linux.go

```diff
@@ -866,14 +768,15 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 
 	sort.Sort(orderedMounts(mounts))
 
+	rspecMount := rspec.Mount{
+		Type:    "bind",
+		Options: nil,
+	}
 	for _, m := range mounts {
-		mnt = rspec.Mount{
-			Type:        "bind",
-			Source:      m.Source,
-			Destination: m.Destination,
-			Options:     append(m.Options, "bind"),
-		}
-		specgen.AddMount(mnt)
+		rspecMount.Destination = m.Destination
```

Comment:
The `rspec.Mount` values look the same as it was being done before in lines 870 to 876, why not leave it as it was?

Reply:
I'm a little confused as to what you mean ?

Reply:
I'm a little confused as to what you mean?

Reply:
It was initially 
```
mnt = rspec.Mount{
			Type:        "bind",
			Source:      m.Source,
			Destination: m.Destination,
			Options:     append(m.Options, "bind"),
		}
specgen.AddMount(mnt)
```

so why not just modify it to be 
```
ctr.SpecAddMount(rspec.Mount{
			Type:        "bind",
			Source:      m.Source,
			Destination: m.Destination,
			Options:     append(m.Options, "bind"),
		})
```
Instead of doing what you have right now
```
rspecMount := rspec.Mount{
		Type:    "bind",
		Options: nil,
	}
...
rspecMount.Destination = m.Destination
rspecMount.Source = m.Source
rspecMount.Options = append(m.Options, "bind")
ctr.SpecAddMount(rspecMount)
```

Reply:
fixed!